### PR TITLE
Update deps/languages

### PR DESCRIPTION
**Pull request by paulswartz**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -42,20 +42,19 @@ defmodule Delta.MixProject do
 
   defp deps do
     [
-      {:gen_stage, "~> 1.0"},
-      {:hackney, "~> 1.15"},
-      {:cowlib, "~> 2.8"},
+      {:gen_stage, github: "paulswartz/gen_stage", branch: "consumer-supervisor-spec"},
```
Comment (digitalcora) [inline mix.exs:45]: Can we add a comment here indicating when we can switch back to the official package? Maybe just a link to your PR would be fine, or a link to the commit it introduced to `master`. (also, nice, José-approved!)

File: mix.exs [hunk 2]
```diff
@@ -5,14 +5,14 @@ defmodule Delta.MixProject do
...
-      elixir: "~> 1.9",
+      elixir: "~> 1.11",
       elixirc_paths: elixirc_paths(Mix.env()),
       compilers: [:phoenix] ++ Mix.compilers(),
       start_permanent: Mix.env() == :prod,
       deps: deps(),
       releases: releases(),
       dialyzer: [
-        plt_add_deps: :transitive,
+        plt_add_deps: :app_tree,
```
Comment (digitalcora) [inline mix.exs:15]: For this project, what's the difference between these options? I'm not sure how this relates to the commit that introduces it, which is the "update Elixir/Erlang" one.

Comment (paulswartz) [reply mix.exs:15]: There's a warning that started appearing (or at least, I started noticing it):
```
Dialyxir has deprecated plt_add_deps: :transitive in favor of app_tree, which includes only runtime dependencies.
```

Comment (digitalcora) [reply mix.exs:15]: Ah, nice, makes sense then!

**Review by digitalcora — APPROVED** [review]
🍰🍰🍰

**Pull request merged**